HULKs/hulk
# Mujoco Walking Engine

## Why? What?
Integrates the walking engine implemented in python into mujoco. Based on #1544 

Fixes #

## ToDo / Known Issues

*If this is a WIP describe which problems are to be fixed.*

## Ideas for Next Iterations (Not This PR)

*If there are some improvements that could be done in a next iteration, describe them here.*

## How to Test

- Run `uv run scripts/mujoco-walking.py`, observe the Nao walking in the simulation
